buildSrc/src/main/kotlin/source-sets-conventions.gradle.kts

```diff
@@ -68,7 +68,10 @@ kotlin {
             progressiveMode = true
 
             optIn("kotlin.ExperimentalMultiplatform")
+            optIn("kotlin.ExperimentalSubclassOptIn")
+            optIn("kotlin.ExperimentalStdlibApi")
```

Comment:
Unused?

Reply:
Hm, maybe I intended for it to be placed in tests, but it seems there is no effect from it for now